darts/aws.yaml

```diff
@@ -77,6 +77,10 @@ chart_variables:
 #  rancher_image_override: rancher/rancher
 #  rancher_image_tag_override: v2.8.6-debug-1
 
+# Set arbitrary helm values (in yaml format) for installing Rancher
+#  values: |
```

Comment:
Please rename this to `rancher_values`, as we install multiple charts and this option is specifically for the Rancher (Manager) installation.

Reply:
Makes total sense, thanks